scripts/install_deps.py

```diff
@@ -0,0 +1,112 @@
+import os
+import platform
+import shutil
+import subprocess
+import sys
+
+from contextlib import contextmanager
+from utils import _virtualenv_enabled
+
+INSTALL_ARGS = "--no-binary :all: --no-build-isolation --no-cache-dir --no-index "
+
+
+class BadRCError(Exception):
+    pass
+
+
+class InstallationError(Exception):
+    pass
+
+
+@contextmanager
+def cd(dirname):
+    original = os.getcwd()
+    os.chdir(dirname)
+    try:
+        yield
+    finally:
+        os.chdir(original)
+
+
+def run(cmd):
+    sys.stdout.write("Running cmd: %s\n" % cmd)
+    p = subprocess.Popen(
+        cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE
+    )
+    stdout, stderr = p.communicate()
+    if p.returncode != 0:
+        output = (stdout + stderr).decode("utf-8")
+        raise BadRCError("Bad rc (%s) for cmd '%s': %s" % (p.returncode, cmd, output))
+    return stdout
+
+
+def _get_package_tarball(package_dir, package_prefix):
+    package_filenames = sorted(
+        [p for p in os.listdir(package_dir) if p.startswith(package_prefix)]
+    )
+    if len(package_filenames) == 0:
+        raise InstallationError(
+            "Unable to find local package starting with %s prefix."
+            % package_prefix
+        )
+    return package_filenames[-1]
```

Comment:
How come we are getting the last one from the list?

Reply:
This evolved from one of our other existing install scripts that was doing it this way. It looks like it was a 2.6 workaround James did that never got cleaned up. We can change to `0` without issue.